### PR TITLE
Merge stable into master

### PR DESCRIPTION
**Pull request by fppt**
# Why is this PR needed?

To sort out branch histories

# What does the PR do?

Merge stable into master

# Does it break backwards compatibility?

No

# List of future improvements not on this PR

N/A

### CODE REVIEW

File: grakn-client/src/main/java/ai/grakn/client/Client.java [hunk 1]
```diff
@@ -18,103 +18,24 @@
...
-        String host = properties.getProperty(GraknConfigKey.SERVER_HOST_NAME.name());
-        int port = Integer.parseInt(properties.getProperty(GraknConfigKey.SERVER_PORT.name()));
-        if (serverIsRunning(new SimpleURI(host, port))) {
-            LOG.info("Server " + host + ":" + port + " is running");
-            return EngineStatus.Running;
-        } else {
-            LOG.info("Server " + host + ":" + port + " is not running");
-            return EngineStatus.NotRunning;
-        }
-    }
```
Comment (aelred) [inline grakn-client/src/main/java/ai/grakn/client/Client.java]: For some reason most of this stuff has not been removed in the final diff :(

Comment (fppt) [reply grakn-client/src/main/java/ai/grakn/client/Client.java]: @aelred wait more of this needs to die?

Comment (aelred) [reply grakn-client/src/main/java/ai/grakn/client/Client.java]: If you look at the changes from all commits, `Client#main` is still there for example. But I removed it in my PR...

**Comment by fppt** [comment]
@aelred re-review please?

**Pull request merged**